crates/task-impls/src/network.rs

```diff
@@ -363,7 +360,7 @@ impl<
     /// which will be used to create a message and transmit on the wire.
     /// Returns `None` if the parsing result should not be sent on the wire.
     /// Handles the `VidDisperseSend` event separately using a helper method.
-    #[allow(clippy::too_many_lines)] // TODO https://github.com/EspressoSystems/HotShot/issues/1704
+    #[allow(clippy::too_many_lines)]
```

Comment:
This issue is just marking that this is a too_many_lines function, but we have several of these in the codebase, and the refactor is painful. Having a bespoke issue for each instance seems like overkill. These are easily "not correct", so they can be fixed whenever it's necessary. Therefore, having an issue seems to be making no material difference in team awareness of the issue. Furthermore, since it's one of our trademark *enormous match statements* it's actually quite easy to follow despite it's size.